eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -209,7 +206,7 @@ jobs:
         ToxEnvParallel: ${{ parameters.ToxEnvParallel }}
         InjectedPackages: $(InjectedPackages)
         BeforeTestSteps:
-          - task: DownloadPipelineArtifact@0
+          - task: DownloadBuildArtifacts@0
```

Comment:
Why the switch from pipeline to build artifacts? @mitchdenny can correct me but I thought pipeline was the newer one which most folks should be using.

Reply:
We were already publishing BuildArtifacts. I simply updated everything to align so we weren't getting warnings for differing artifact types.

Reply:
In a followup PR I'll update everything to PipelineArtifacts (publish side as well).

Reply:
K I've got a[ commit ready to go]()https://github.com/scbedd/azure-sdk-for-python/commit/f752a336d4f610fb8c300cb3b9fdecaf614e868b in that cleans that up + also removes the pending cleanup items.

Reply:
I just merged that commit. Will do another test run over the weekend to confirm.